pkg/cmd/grafana-server/commands/cli.go

```diff
@@ -111,6 +109,9 @@ func RunServer(opts ServerOptions) error {
 		return err
 	}
 
+	setBuildInfo(metrics.ProvideRegisterer(cfg), opts)
+	checkPrivileges()
+
```

Comment:
@toddtreece this moves the setBuildInfo to after Cfg is created (setting.NewCfgFromArgs) leading to empty Cfg.BuildVersion , leading to wrong CDN url (and no assets loaded)